source/includes/_wallet.md

```diff
@@ -1 +1,995 @@
+
+## Create New Wallet
+
+```javascript
+let id, witness;
+```
+
+```shell--vars
+id = 'foo'
+witness = false
+```
+
+```shell--curl
+curl $url/wallet/$id \
+  -X PUT \
+  --data '{"witness":'$witness'}'
+```
+
+```shell--cli
+bcoin cli wallet create $id --witness=$witness
+```
+
+```javascript
+const client = new bcoin.http.Client();
+const options = {
+  id: id,
+  witness: witness
+};
+
+(async() => {
+  const newWallet = await client.createWallet(options)
+})();
+```
+
+> Sample response:
+
+```json
+{
+  "network": "testnet",
+  "wid": 2,
+  "id": "foo",
+  "initialized": true,
+  "watchOnly": false,
+  "accountDepth": 1,
+  "token": "d9de1ddc83bf058d14520a203df6ade0dc92a684aebfac57b667705b4cac3916",
+  "tokenDepth": 0,
+  "state": {
+    "tx": 0,
+    "coin": 0,
+    "unconfirmed": "0.0",
+    "confirmed": "0.0"
+  },
+  "master": {
+    "encrypted": false
+  },
+  "account": {
+    "name": "default",
+    "initialized": true,
+    "witness": false,
+    "watchOnly": false,
+    "type": "pubkeyhash",
+    "m": 1,
+    "n": 1,
+    "accountIndex": 0,
+    "receiveDepth": 1,
+    "changeDepth": 1,
+    "nestedDepth": 0,
+    "lookahead": 10,
+    "receiveAddress": "muYkrSDbD8UhyWBMXxXf99EKWn22YqmwyF",
+    "nestedAddress": null,
+    "changeAddress": "mwveV7A6svE5EGGSduZmMKTwcbE775NVFt",
+    "accountKey": "tpubDDh2XgSds1vBbeVgye88gsGQeCityoywRndtyrXcmvWqCgsFUyUKwzeDv8HiJhu9fC8jRAFMqxr4jj8eRTNTycmMao5wmsAScVf4jSMdPYZ",
+    "keys": []
+  }
+}
+```
+
+Create a new wallet with a specified ID.
+
+### HTTP Request
+
+`PUT /wallet/:id`
+
+Parameters | Description
+---------- | -----------
+id <br> _string_ | id of wallet you would like to create
+
+<aside class="notice">
+See <a href="#wallet-options">Wallet Options</a> for full list and description of possible options that can be passed
+</aside>
+
+
+## Change Passphrase
+```javascript
+let id, oldPass, newPass;
+```
+
+```shell--vars
+id='foo'
+oldPass = 'oldpass123'
+newPass='newpass123'
+```
+
+```shell--cli
+> No command available
+```
+
+```shell-curl
+curl $url/wallet/$id/passphrase \
+  -X POST
+  --data '{"old":'$oldPass', "new":'$newPass'}'
+```
+
+```javascript
+const client = new bcoin.http.CLient();
+
+(async () => {
+  const response = await client.setPassphrase(id, oldPass, newPass);
+  console.log(response);
+});
+```
+
+> Sample Response:
+
+```json
+{"success": true}
+```
+
+Change wallet passphrase. Encrypt if unencrypted.
+
+### HTTP Request
+
+`POST /wallet/:id/passphrase`
+
+### Body Parameters
+Paramters | Description
+--------- | ---------------------
+old <br> _string_ | Old passphrase. Pass in empty string if none
+passphrase <br> _string | New passphrase
+
+<aside class="notice">
+  Note that the old passphrase is still required even if none was set prior. In this case, an empty string should be passed for the old passphrase.
+  e.g. <code>client.setPassphrase(id,'""', 'newPass')</code>
+</aside>
```

Comment:
Yeah there are some others that will change from a PR I have filed too. I'm wondering how to deal with that since we don't know when they'll get in or if they'll change.

Reply:
Thanks yeah it was not perfect, will review the PR later.